dist/Locale-Maketext/Makefile.PL

```diff
@@ -28,6 +28,10 @@ WriteMakefile(
 );
 
 sub MY::postamble {
+    # .PHONY is not portable
+    my $self = shift;
+    return '' if $self->{PERL_CORE};
+
```

Comment:
I'd go for something more like:
```perl
my $self = shift;
# .PHONY is not portable
my $phony_line = $self->can('is_make_type') && $self->is_make_type('gmake')
    ? '.PHONY: tags critic'
    : '';
return "$phony_line\n\n" . <<'MAKE_FRAG';
tags:
...
```

Reply:
I thought about something like that, but how can we be sure only GNU make supports it?  At least [BSD make](https://wiki.netbsd.org/tutorials/bsd_make/) claims to have .PHONY as well.

Reply:
For one, I think false negatives are OK here. These extra targets are only relevant for developers (of the module), not people just installing off CPAN.

And we can easily extend the test:
```perl
my $phony_line = $self->can('is_make_type') && ($self->is_make_type('gmake') || $self->is_make_type('bsdmake'))
...
```

Reply:
thanks for the feedback -- I've followed your suggestion